fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -13,79 +13,39 @@
 Possible areas of improvement
   1. Save to calling directory, not project directory.
 """
+import math
 import os
 import sys
-from typing import Dict, List, Optional, Union
-import math
-import pandas as pd
 import time
+from datetime import datetime, date
+from typing import Dict, List, Optional, Union
+
 import zulip
-from datetime import datetime,date
+import pandas as pd
+
+try:
+    from fhir_zulip_nlp.google_sheets import get_sheets_data
+except ModuleNotFoundError:
+    from google_sheets import get_sheets_data
 
 
 # Vars
 PKG_DIR = os.path.dirname(os.path.realpath(__file__))
 PROJECT_DIR = os.path.join(PKG_DIR, '..')
+ENV_DIR = os.path.join(PROJECT_DIR, 'env')
```

Comment:
New directory that you should make at the root of the cloned repository. I'll send you this folder and you can unzip it there.